### PR TITLE
started second function

### PR DESCRIPTION
**Pull request by maryrsmcphee**
just a start, I think it recurses infinitely... 
I did the comparisons and I think they're right but I don't think the base case/return is right if you could check that out?

### CODE REVIEW

File: PracticeExercise2/main.cpp [hunk 1]
```diff
@@ -82,10 +82,19 @@ int find_max_sum_of_nodes (BinaryTreeNode* T, int &temp_max_sum) {
     int left_sum = find_max_sum_of_nodes(T->left, temp_max_sum);
```
Comment (SammyRobensParadise) [inline PracticeExercise2/main.cpp:85]: Because these happen before any of the logic it is just going to recur forever

Comment (SammyRobensParadise) [reply PracticeExercise2/main.cpp:85]: you need to add in the summation is until a condition is met

Comment (maryrsmcphee) [reply PracticeExercise2/main.cpp:85]: ohhhhh right

Comment (SammyRobensParadise) [reply PracticeExercise2/main.cpp:85]: also need to check if null or smth

Comment (maryrsmcphee) [reply PracticeExercise2/main.cpp:85]: wait why is that in the starter code then?

Comment (SammyRobensParadise) [reply PracticeExercise2/main.cpp:85]: hmm idk.. but I think the way its structures its just going to loop that first part forever

Comment (maryrsmcphee) [reply PracticeExercise2/main.cpp:85]: ok with the check at the top it should stop once it reaches the bottom

**Review by SammyRobensParadise — APPROVED** [review]

**Pull request merged**